wpilibj/src/main/java/edu/wpi/first/wpilibj/simulation/FlywheelSim.java

```diff
@@ -21,7 +21,7 @@
 
 /** Represents a simulated flywheel mechanism. */
 public class FlywheelSim extends LinearSystemSim<N1, N1, N1> {
-  // Gearbox for the flywheel.
+  // GearBox for the flywheel.
```

Comment:
This change was the opposite of what I requested. I was referring to how getGearBox() should be getGearbox() instead.
```suggestion
  // Gearbox for the flywheel.
```

Reply:
Sorry about that.